assets/css/media.css

```diff
@@ -0,0 +1,120 @@
+@media (min-width: 768px) {
+  #a-login,
+  #a-login-2 {
+    font-size: 18px;
+    text-decoration: none;
+  }
+
+  #subs-btn,
+  #subs-btn-2 {
+    font-size: 15px;
+  }
+
+  #nav-menu {
+    font-size: 26px;
+  }
+
+  .opinion {
+    border-bottom: 0 solid !important;
```

Comment:
Don't use !important

Reply:
Removed the !important from the .css files.